src/errors.rs

```diff
@@ -249,18 +245,31 @@ impl Errors {
         // Log the primary pointer:
         self.log_pointer_primary(report.primary(), report.indentation(), report.lvl.severity);
         // Log the other pointers:
-        report.pointers.iter().skip(1).for_each(|pointer| {
+        report.pointers.iter().enumerate().skip(1).for_each(|(index, pointer)| {
```

Comment:
This might be more idiomatic with slice.windows() :)
https://doc.rust-lang.org/std/primitive.slice.html#method.windows

Reply:
Amazing, I had no idea this existed.